src/main/kotlin/org/jetbrains/research/ml/ast/transformations/PyElementGeneratorExt.kt

```diff
@@ -0,0 +1,21 @@
+package org.jetbrains.research.ml.ast.transformations
+
+import com.jetbrains.python.psi.LanguageLevel
+import com.jetbrains.python.psi.PyBoolLiteralExpression
+import com.jetbrains.python.psi.PyElementGenerator
+import com.jetbrains.python.psi.PyExpression
+
+private val defaultLanguageLevel = LanguageLevel.getDefault()
+
+fun PyElementGenerator.createBoolLiteralExpression(value: Boolean): PyBoolLiteralExpression {
+    return createExpressionFromText(defaultLanguageLevel, if (value) "True" else "False") as PyBoolLiteralExpression
```

Comment:
This if expression looks very strange. Why can not we use `str(value)`?

Reply:
We need to convert a (kotlin) boolean value to the corresponding Python literal, I'm not sure how `str(value)` is related to this...

Reply:
Oh sorry, it is my mistake, of course, you are right